EIPS/eip-3386.md

```diff
@@ -0,0 +1,315 @@
+---
+eip: 3386
+title: ERC-721 and ERC-1155 to ERC-20 Wrapper
+author: Calvin Koder <ashrowz@mailbox.org>
```

Comment:
```suggestion
author: Calvin Koder (@ashrowz)
```
It is strongly encouraged to have at least one author with a github account listed so they can approve changes to the EIP.